frame/election-provider-multi-phase/src/benchmarking.rs

```diff
@@ -278,7 +278,8 @@ frame_benchmarking::benchmarks! {
 		let witness = SolutionOrSnapshotSize { voters: v, targets: t };
 		let raw_solution = solution_with_size::<T>(witness, a, d)?;
 		let ready_solution =
-			<MultiPhase<T>>::feasibility_check(raw_solution, ElectionCompute::Signed)?;
+			<MultiPhase<T>>::feasibility_check(raw_solution, ElectionCompute::Signed)
+				.map_err(|e| -> &'static str { e.into() } )?;
```

Comment:
```suggestion
				.map_err(<&str>::from)?;
```

Tiny nit. Writing out the closure signature just seems wrong to my brain 😁